### PR TITLE
Adds /cla page

### PR DESCRIPTION
**Pull request by shawyu-okta**
## Description:
- **What's changed?** 
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-362122](https://oktainc.atlassian.net/browse/OKTA-362122)

### CODE REVIEW

**Comment by jakubvul-okta** [comment]
Ack

File: packages/@okta/vuepress-site/cla/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - The CLA is used to protect the rights of you the Contributor, Okta, and Okta’s users. The CLA does not take away any right, title, or interest you have in your contribution and only assigns the rights specified in the CLA to Okta in order to allow Okta and Okta’s users to use your contribution.
+    - "You can download our CLA here: <a href='https://developer.okta.com/sites/all/themes/developer/pdf/okta_individual_contributor_license_agreement_2016-11.pdf'>Okta Individual Contributor License Agreement</a>"
```
Comment (francoislehoux-okta) [inline packages/@okta/vuepress-site/cla/index.md:15]: Where is this hosted at, drupal?

File: packages/@okta/vuepress-site/cla/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - "A Contributor License Agreement would be required for the following as they are not Obvious Fixes:"
+    - - Anything that results in a change in functionality
+      - A new feature
+      - A translation, or a refactor that changes functionality
+      - Extensive or creative comments, changes to README files or documentation
+
+  - heading: How Do I Submit an Obvious Fix?
+    tag: h3
+    paragraphs:
+    - "If you believe your contribution is an Obvious Fix, please add “Obvious Fix” to the pull request description. If Okta believes it is not an Obvious Fix we may request that you <a href='https://developer.okta.com/cla/okta_individual_contributor_license_agreement_2016-11.pdf'>sign a CLA</a> before we merge your pull request."
```
Comment (francoislehoux-okta) [inline packages/@okta/vuepress-site/cla/index.md:47]: Where is this hosted at, drupal?

Comment (shawyu-okta) [reply packages/@okta/vuepress-site/cla/index.md:47]: Yeah, good catch, I will pull this into this site as a static file

File: packages/@okta/vuepress-site/cla/index.md (hunk 1 above)
Comment (shawyu-okta) [reply packages/@okta/vuepress-site/cla/index.md:15]: This is hosted on the old Drupal site -- I'll also port this file over and update the URL

File: packages/@okta/vuepress-site/cla/index.md (hunk 2 above)
Comment (shawyu-okta) [reply packages/@okta/vuepress-site/cla/index.md:47]: Actually I just downloaded the two file and they appear to be the same file hosted in different places, so I'll just bring one over and update this link.

**Review by shawyu-okta — COMMENTED** [review]
@francoislehoux-okta links updated to reference local static file

**Review by francoislehoux-okta — APPROVED** [review]
LGTM

**Pull request merged**